File: backend/api_company/serializers/project_task.py
from rest_framework import serializers
from api_company.models import ProjectTask
from api_company.serializers import MemberSerializer


class ProjectTaskSerializer(serializers.ModelSerializer):

    member_list = serializers.SerializerMethodField()

    def get_member_list(self, obj):
        members = obj.members.all()
        serializer = MemberSerializer(members, many=True)
        return serializer.data

    class Meta:
        model = ProjectTask
        fields = ('id', 'title', 'members', 'member_list', 'user_id', 'project', 'description', 'start_time', 'end_time', 'status')

File: frontend/src/app/models/project-task.ts
import {Member} from "./member";

export class ProjectTask {
    id: number;
    title: string;
    description: string;
    members: any;
    // tslint:disable-next-line:variable-name
    member_list: Member[];
    status: string;
    // tslint:disable-next-line:variable-name
    start_time: any;
    // tslint:disable-next-line:variable-name
    end_time: any;
}
